src/vm/ceemain.cpp

```diff
@@ -1760,13 +1728,6 @@ void STDMETHODCALLTYPE EEShutDownHelper(BOOL fIsDllUnloading)
                 Interpreter::Terminate();
 #endif // FEATURE_INTERPRETER
 
-#ifdef SHOULD_WE_CLEANUP
```

Comment:
There is also a `SHOULD_WE_CLEANUP` in https://github.com/dotnet/coreclr/blob/master/src/utilcode/posterror.cpp#L31-L39. Perhaps we should get rid of both of them?